### PR TITLE
Add JS functions compat data

### PR DESCRIPTION
**Pull request by Elchi3**

### CODE REVIEW

File: javascript/builtins/Function.json [hunk 1]
```diff
@@ -0,0 +1,1032 @@
...
+                "version_added": null
+              },
+              "edge": {
+                "version_added": null
+              },
+              "edge_mobile": {
+                "version_added": null
+              },
+              "firefox": {
+                "version_added": null
```
Comment (wbamberg) [inline javascript/builtins/Function.json]: 13 according to the page

File: javascript/builtins/Function.json [hunk 2]
```diff
@@ -0,0 +1,1032 @@
...
+              "edge_mobile": {
+                "version_added": true
+              },
+              "firefox": {
+                "version_added": true
+              },
+              "firefox_android": {
+                "version_added": true
+              },
+              "ie": {
```
Comment (wbamberg) [inline javascript/builtins/Function.json]: 12 according to the page

File: javascript/builtins/Function.json [hunk 3]
```diff
@@ -0,0 +1,1032 @@
...
+                "version_added": false
+              },
+              "ie_mobile": {
+                "version_added": false
```
Comment (wbamberg) [inline javascript/builtins/Function.json]: 12 according to the page

File: javascript/builtins/Function.json [hunk 4]
```diff
@@ -0,0 +1,1032 @@
...
+              "description": "Configurable: true",
+              "support": {
+                "webview_android": {
+                  "version_added": null
+                },
+                "chrome": {
+                  "version_added": "43"
+                },
+                "chrome_android": {
+                  "version_added": null
```
Comment (wbamberg) [inline javascript/builtins/Function.json]: should be the same as Chrome Desktop? not sure.

File: javascript/builtins/Function.json [hunk 5]
```diff
@@ -0,0 +1,1032 @@
...
+            },
+            "status": {
+              "experimental": false,
+              "standard_track": true,
+              "deprecated": false
+            }
+          },
+          "generic_arrays_as_arguments": {
+            "__compat": {
+              "description": "ES 5.1: generic array-like object as arguments",
```
Comment (wbamberg) [inline javascript/builtins/Function.json]: Could enclose argments in \<code\> to show it's a keyword.

File: javascript/builtins/Function.json [hunk 6]
```diff
@@ -0,0 +1,1032 @@
...
+              "support": {
+                "webview_android": {
+                  "version_added": null
+                },
+                "chrome": {
+                  "version_added": true
+                },
+                "chrome_android": {
+                  "version_added": null
```
Comment (wbamberg) [inline javascript/builtins/Function.json]: same as Chrome Desktop?

File: javascript/builtins/Function.json [hunk 7]
```diff
@@ -0,0 +1,1032 @@
...
+                },
+                "edge_mobile": {
+                  "version_added": null
+                },
+                "firefox": {
+                  "version_added": "4"
+                },
+                "firefox_android": {
+                  "version_added": "37"
```
Comment (wbamberg) [inline javascript/builtins/Function.json]: 4 according to the page

File: javascript/builtins/Function.json [hunk 8]
```diff
@@ -0,0 +1,1032 @@
...
+              "edge_mobile": {
+                "version_added": false
+              },
+              "firefox": {
+                "version_added": true
+              },
+              "firefox_android": {
+                "version_added": true
+              },
+              "ie": {
```
Comment (wbamberg) [inline javascript/builtins/Function.json]: ? in the page (and for some other browsers here too)

File: javascript/builtins/Function.json [hunk 9]
```diff
@@ -0,0 +1,1032 @@
...
+              "deprecated": false
+            }
+          }
+        },
+        "toString": {
+          "__compat": {
+            "mdn_url": "https://developer.mozilla.org/docs/Web/JavaScript/Reference/Global_Objects/Function/toString",
+            "support": {
+              "webview_android": {
+                "version_added": false
```
Comment (wbamberg) [inline javascript/builtins/Function.json]: Yes in the page (and for other browsers)

File: javascript/builtins/Function.json [hunk 10]
```diff
@@ -0,0 +1,1032 @@
...
+              "safari": {
+                "version_added": false
+              },
+              "safari_ios": {
+                "version_added": false
+              }
+            },
+            "status": {
+              "experimental": false,
+              "standard_track": false,
```
Comment (wbamberg) [inline javascript/builtins/Function.json]: true?

File: javascript/builtins/Function.json [hunk 11]
```diff
@@ -0,0 +1,1032 @@
...
+                "safari": {
+                  "version_added": false
+                },
+                "safari_ios": {
+                  "version_added": false
+                }
+              },
+              "status": {
+                "experimental": false,
+                "standard_track": false,
```
Comment (wbamberg) [inline javascript/builtins/Function.json]: true?

File: javascript/builtins/Generator.json [hunk 12]
```diff
@@ -0,0 +1,222 @@
...
+              "version_added": false
+            },
+            "ie_mobile": {
+              "version_added": false
+            },
+            "nodejs": {
+              "version_added": true
+            },
+            "opera": {
+              "version_added": true
```
Comment (wbamberg) [inline javascript/builtins/Generator.json]: no in the page (also for mobile)

File: javascript/builtins/Generator.json [hunk 13]
```diff
@@ -0,0 +1,222 @@
...
+            },
+            "opera_android": {
+              "version_added": true
+            },
+            "safari": {
+              "version_added": "10"
+            },
+            "safari_ios": {
+              "version_added": "10"
```
Comment (wbamberg) [inline javascript/builtins/Generator.json]: no in the page (also for mobile)

File: javascript/builtins/AsyncFunction.json [hunk 14]
```diff
@@ -0,0 +1,114 @@
+{
+  "javascript": {
+    "builtins": {
+      "AsyncFunction": {
+        "__compat": {
+          "mdn_url": "https://developer.mozilla.org/docs/Web/JavaScript/Reference/Global_Objects/AsyncFunction",
+          "support": {
+            "webview_android": {
+              "version_added": null
+            },
```
Comment (jpmedley) [inline javascript/builtins/AsyncFunction.json]: webview_android is also 55.

https://www.chromestatus.com/features/5643236399906816

File: javascript/builtins/Function.json (hunk 2 above)
Comment (Elchi3) [reply javascript/builtins/Function.json]: There is no IE 12. This means Edge should be true.

File: javascript/builtins/Function.json (hunk 3 above)
Comment (Elchi3) [reply javascript/builtins/Function.json]: There is no IE 12 mobile. Edge is marked as true.

File: javascript/builtins/Function.json (hunk 8 above)
Comment (Elchi3) [reply javascript/builtins/Function.json]: toSource is a SpiderMonkey specific API. This should be false everywhere except Firefox.

File: javascript/builtins/Generator.json (hunk 12 above)
Comment (Elchi3) [reply javascript/builtins/Generator.json]: Generators are around for a while now. They must have appeared in Opera in some version.

File: javascript/builtins/Generator.json (hunk 13 above)
Comment (Elchi3) [reply javascript/builtins/Generator.json]: Generators are in Safari 10, page isn't up to date.

**Pull request merged**